sources/api/certdog/src/main.rs

```diff
@@ -36,9 +39,9 @@ struct Args {
     #[argh(option, default = "LevelFilter::Info", short = 'l')]
     /// log-level trace|debug|info|warn|error
     log_level: LevelFilter,
-    #[argh(option, default = "constants::API_SOCKET.to_string()", short = 's')]
-    /// socket-path path to apiserver socket
-    socket_path: String,
+    #[argh(option, default = "DEFAULT_CONFIG_FILE.to_string()", short = 'c')]
+    /// config-path path to apiserver socket
+    config_path: String,
```

Comment:
```suggestion
    #[argh(option, default = "DEFAULT_CONFIG_FILE.to_string()", short = 'c')]
    /// config-path path to config file
    config_path: String,
```

Can we use an actual `PathBuf` to store the path or is the `String` helpful for some reason?